src/tasks/components/RunLogRow.tsx

```diff
@@ -41,7 +42,9 @@ class RunLogRow extends PureComponent<Props> {
     }
 
     const newdate = new Date(dt)
-    const formatted = moment(newdate).format(DEFAULT_TIME_FORMAT)
+    const formatted = createDateTimeFormatter(
+      resolveTimeFormat(DEFAULT_TIME_FORMAT)
```

Comment:
I don't think you want to use `resolveTimeFormat` here - that takes the structure in [timeFormat](https://github.com/influxdata/ui/blob/master/src/visualization/utils/timeFormat.ts#L3) and converts it for display in a dropdown. I think passing in `DEFAULT_TIME_FORMAT` which is a string, should work.

```ts
const formatted = createDateTimeFormatter(DEFAULT_TIME_FORMAT).format(newDate)
```

or combining it in one line if you like:

```ts
return createDateTimeFormatter(DEFAULT_TIME_FORMAT).format(new Date(dt))
```